packages/story-editor/src/app/quickActions/utils/getResetProperties.js

```diff
@@ -21,7 +21,7 @@ import { ELEMENT_TYPES } from '@googleforcreators/elements';
 /**
  * Internal dependencies
  */
-import { RESET_PROPERTIES, RESET_DEFAULTS } from '../constants';
+import { RESET_PROPERTIES, RESET_DEFAULTS } from '../../highlights/constants';
```

Comment:
These constants should also be moved to the `quickActions` folder and out of `highlights`